### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by EmiLoKiJu**
- Added functionalities to the ToDoList
- The Objects with description, boolean, and index
- The Design is similar to what is asked
- Used Webpack to mount the project

### CODE REVIEW

**Comment by Davidosky007** [comment]
Hi @EmiLoKiJu 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has node modules pushed to GitHub which is preventing proper review kindly run these commands to remove the folder `git rm -r --cached node_modules` `git commit -am "node_modules be gone!"`
`git push origin master
` Also i noticed the requirements for this project title `To Do list: list structure` are missing if this was a mistake kindly submit the right PR 
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome-books] <a name="about-project"></a>
+
+**[Awesome-books]** is a web site with a RWD.
```
Comment (Kossi-stack) [inline README.md]: - [ ] Kindly write the appropriate name of the project

File: README.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+In order to run this project you need:
+
+A browser with HMTL5 and CSS3
+Install Google Chrome or any browser that supports the above.
+
+### Setup
+
+You don't need to setup this project
+
+### Install
```
Comment (Kossi-stack) [inline README.md:80]: - [ ] Kindly give the procedure on how one can install your project.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @EmiLoKiJu,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by DammyShittu — APPROVED** [review]
Hi @EmiLoKiJu,

## APPROVED 🟢

## Highlights
✔️ Descriptive readme doc

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**